### PR TITLE
Add GitHub Actions workflow for building images

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt install -y libarchive-tools curl fdisk qemu-user-static zstd
+      - name: Build ODroid image
+        run: sudo ./build.sh odroid.img
+      - name: Compress image
+        run: zstd odroid.img
```
Comment (trickeydan) [inline .github/workflows/image.yml:18]: 👍 to zstd

File: stage0/10-bootstrap.sh [hunk 2]
```diff
@@ -19,3 +19,16 @@ cd "$BUILD_DIR/.."
 
 info "Injecting" "bees 🐝🐝🐝🐝"
 cat res/bee >> "$BUILD_DIR/etc/issue"
+
+info "Creating build metadata"
+(
+  echo BUILD_USER="$(whoami)"
+  echo BUILD_HOST="$(hostname)"
+  echo BUILD_TIME="$(date)"
+  echo BUILD_COMMIT="$(git rev-parse --short HEAD)"
```
Comment (trickeydan) [inline stage0/10-bootstrap.sh:28]: Please can we add BUILD_TAG=`git describe --exact-match --tags`?

**Review by trickeydan — APPROVED** [review]
Approved with suggestions. Nice work!

**Comment by raccube** [comment]
Should I maybe also make builds trigger on PRs?

**Comment by trickeydan** [comment]
> Should I maybe also make builds trigger on PRs?

Let's run with this for now. We'll need to fiddle with things once the OS is looking stable

**Pull request merged**